LICENSE

```diff
@@ -59,6 +59,54 @@ direction to make these releases possible.
 B. TERMS AND CONDITIONS FOR ACCESSING OR OTHERWISE USING PYTHON
 ===============================================================
 
+ACTIVESTATE LICENSE AGREEMENT FOR PYTHON 2.7 GREATER THAN 2.7.18
```

Comment:
Is this just renaming the PSF license?

Reply:
Yes, renaming and changing the company name. I wanted the license type to remain identical to python3

Reply:
Ok let me just verify we are ok with effectively a 'custom' OS license name type. Should be ok but will verify.